src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/out/CallbackResultListResponse.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.model.out;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class CallbackResultListResponse {
```

Comment:
why do we need to wrapt the response?

Reply:
I will add `count` to make it more sensible